e2e/files/volume-claim.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
same here, if it's needed to have a separate pod and PVC template we can separate it out in the example folder.